pkg/driverbuilder/builder/builders.go

```diff
@@ -105,6 +103,10 @@ func Script(b Builder, c Config, kr kernelrelease.KernelRelease) (string, error)
 	}
 
 	td := b.TemplateData(c, kr, urls)
+	if tdErr, ok := td.(error); ok {
```

Comment:
TemplateData can now return an interface that resolves to an error, making `Script` exit with an error.